03_Modules/Commands/src/module/module.ts

```diff
@@ -49,6 +53,22 @@ export class CommandsModule implements OcpiModule {
         this.logger,
       ),
     );
+
+    Container.set(
```

Comment:
building on my comment here https://github.com/citrineos/citrineos-core/pull/222#issuecomment-2289740960

i think if you do what's in the linked comment, you shouldn't need to set here anymore! it'll just inject automatically once the server spins up :eyes:

Reply:
Thanks for sharing this!